emcc.py

```diff
@@ -1695,6 +1695,10 @@ def get_final():
             lto_level = options.opt_level
           else:
             lto_level = 0
+          if DEBUG:
+            for i, linker_input in enumerate(linker_inputs):
+              name = 'linker_input_%d_%s' % (i, os.path.basename(linker_input))
+              shutil.copyfile(linker_input, os.path.join(shared.get_emscripten_temp_dir(), name))
```

Comment:
I think linker_inputs might not all be files.. some flags also get passed through as "inputs".  e..g `-Wl,--whole-archive`.  

Also even if we limit it to files won't this be pretty huge as it will copy all of the object files and all the library files?

Maybe thats OK ?  Seem like a lot to me though.

Reply:
Ah, thanks, I see now that it has the linker flags too. So this isn't so easy.

Regarding size, right now in asm2wasm we save `emcc-0-basebc.bc` which is the combined total bitcode input, which is useful for debugging. This would be the closest we can get for the lld path. In both cases the total size saved is equal to the total of all the inputs, which can indeed be large, but this is only in EMCC_DEBUG mode.